cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandSupport.java

```diff
@@ -11,330 +9,525 @@
+        probe.setInitialDelaySeconds(60);
+        probe.setPeriodSeconds(10);
+        probe.setSuccessThreshold(1);
+        probe.setTimeoutSeconds(5);
+        probe.setHttpGet(new HTTPGetAction(null, null, "/", new IntOrString("rest-api"), "HTTP"));
 
-        return status;
+        return probe;
     }
 
-    public static Map<String, Object> createConfig(DebeziumOperandConfiguration configuration, ObjectNode connectorSpec) {
-        Map<String, Object> config = new TreeMap<>();
+    public static String jaasConfig(ServiceAccountSpec sa) {
 
-        // add external configuration
-        if (configuration.kafkaConnector().config() != null) {
-            config.putAll(configuration.kafkaConnector().config());
-        }
+        return String.format(
+            "org.apache.kafka.common.security.plain.PlainLoginModule required username=\"%s\" password=\"%s\";",
+            sa.getClientId(),
+            new String(Base64.getDecoder().decode(sa.getClientSecret()), StandardCharsets.UTF_8));
+    }
 
-        if (connectorSpec != null) {
-            var cit = connectorSpec.fields();
-            while (cit.hasNext()) {
-                final var property = cit.next();
+    public static Map<String, String> createSchemaHistoryConfig(ManagedConnector connector, ServiceAccountSpec sa) {
+        return Map.of(
+            "schema.history.internal.kafka.bootstrap.servers", connector.getSpec().getDeployment().getKafka().getUrl(),
+            "schema.history.internal.kafka.topic", connector.getMetadata().getName() + "-schema-history",
 
-                if (!property.getValue().isObject()) {
-                    config.put(
-                        property.getKey(),
-                        property.getValue().asText());
-                } else {
-                    config.putIfAbsent(
-                        property.getKey(),
-                        "${file:/opt/kafka/external-configuration/"
-                            + EXTERNAL_CONFIG_DIRECTORY
-                            + "/"
-                            + EXTERNAL_CONFIG_FILE
-                            + ":" + property.getKey() + "}");
-                }
-            }
+            "schema.history.internal.producer.security.protocol", "SASL_SSL",
+            "schema.history.internal.producer.sasl.mechanism", "PLAIN",
+            "schema.history.internal.producer.sasl.jaas.config", jaasConfig(sa),
+
+            "schema.history.internal.consumer.security.protocol", "SASL_SSL",
+            "schema.history.internal.consumer.sasl.mechanism", "PLAIN",
+            "schema.history.internal.consumer.sasl.jaas.config", jaasConfig(sa));
+    }
+
+    public static ResourceRequirements resources() {
+        return new ResourceRequirementsBuilder()
+            .addToRequests("cpu", new Quantity("10m"))
+            .addToRequests("memory", new Quantity("256Mi"))
+            .addToLimits("cpu", new Quantity("500m"))
+            .addToLimits("memory", new Quantity("1Gi"))
```

Comment:
Is this still valid: https://web.archive.org/web/20220805232857/https://home.robusta.dev/blog/stop-using-cpu-limits/ ?
And didn't we agree to not use Burstable (memory) pods that have memory requests and limits set to the same value?

Reply:
I took it from the[ current implementation ](https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/blame/a6dc859aa51b12473ad570dc383ff73442ecbad7/cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandController.java#L227-L232)